content/en/docs/concepts/cluster-administration/api-priority-and-fairness.md

```diff
@@ -0,0 +1,125 @@
+---
+title: API priority and fairness
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each apiserver
+applies some prioritization and fairness in its handling of requests.
+This includes not only kube-apiservers but aggregated apiservers too.
+Prioritization and fairness applies only to requests that are not
+_long running_.  Long running requests are WATCHes and EXECUTEs.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Status
+
+This feature is named "RequestManagement" and is alpha in
+release 1.16.  As an alpha feature, it is disabled by default.  When
+enabled it replaces a request filter that enforces simple aggregate
+concurrency limits with this feature, which can be seen as a more
+general version of that earlier functionality.
+
+## Outline
+
+Each apiserver independently applies prioritization and fairness,
+based on API objects that configure this feature.  This involves two
+concepts: priority level and flow schema.
+
+### Priority level
+
+Each apiserver has an aggregate concurrency limit and divides
+it&emdash;without examining the request load&emdash;among several
+priority levels.  The priority levels operate independently of each
+other.  That is, the apiserver classifies a given request to exactly
+one priority level and, for a given priority level, handles at most
+the corresponding number of requests at any given time.  One special
+priority level&emdash;used for meta-requests and other top-priority
+traffic&emdash;imposes no limit.  This priority level and the requests
+mapped to it are described as "exempt", because of the lack of
+concurrency limitation.
+
+For a non-exempt request, if it is not allowed to be handled
+immediately then the apiserver will either reject the request or hold
+it in a queue for a while.  Each non-exempt priority level has a
+number of FIFO queues.  Whenever that priority level's number of
+requests currently being handled drops below the concurrency limit,
+the apiserver draws requests to begin handling from these queues using
+a technique that is an appropriate generalization of
+[fair queuing](https://en.wikipedia.org/wiki/Fair_queuing).
```

Comment:
I'd recommend leaving out the link and discussion of fair queuing. Adding that here is prone to doc-rot once (as is inevitable) someone swaps in a different algorithm for handling the queues. I'd suggest shortening to just this:

```
For a non-exempt request, if it is not allowed to be handled immediately then
the apiserver will either reject the request or hold it in a [queue](#Queuing) for
later processing.
```